### ISSUE TITLE
1.6.0

### ISSUE THREAD

**Issue by az3r0th23** [kiideveloper/pokiimap#87]
app becomes laggy after the update

**Comment by az3r0th23** [comment]
too much freeze

**Comment by thebigsmileXD** [comment]
i confirm, it lags like idk. and goddamn banner ads, i go back to 1.4

**Comment by notmyusualname** [comment]
Agreed. I'd be happy to pay for the ads to disappear. I even tried to "buy you a beer" but paypal says you're not taking money right now...which is odd. The banner ad really gets in the way, pops up after you close it and slows the app down too much. I love this app. It's my all time favourite app. I want to give you some money for your excellence. I also want to be ad free.

**Comment by Googulator** [comment]
Where are the sources for 1.6.0? I would like to make a pull request for Samsung multi-window / pop-up view support, but it seems the source code in the repository has not been updated for months.

The LICENSE file says GPL, and the docs explicitly say it's open-source and forks/pull requests are welcome.
